src/Provider/AbstractProvider.php

```diff
@@ -126,7 +138,7 @@ public function __construct(array $options = [], array $collaborators = [])
      * Sets the grant factory instance.
      *
      * @param  GrantFactory $factory
-     * @return $this
+     * @return self
```

Comment:
Is this the standard way to hint this? I've been using `$this` for years... doing it wrong?